ros_ws/src/car_control/src/dms_controller.cpp

```diff
@@ -11,6 +11,9 @@ DMSController::DMSController()
         this->m_node_handle.subscribe<std_msgs::Int64>(TOPIC_DMS, 1, &DMSController::dmsCallback, this);
 
     this->m_command_pulisher = this->m_node_handle.advertise<std_msgs::String>(TOPIC_COMMAND, 1);
+
+    m_node_handle.getParam(DMS_CHECK_RATE_PARAMETER, this->dms_check_rate);
+    m_node_handle.getParam(DMS_EXPIRATION_PARAMETER, this->dms_expiration);
```

Comment:
**MAY**
Add a sanity check e.g.
- Is any value zero or below?
- In any value way to high?